docs/settingUp.md

```diff
@@ -88,9 +88,14 @@ Eclipse IDE is our preferred development environment. Support requests related t
       ![devverification-guide-1.png](images/devverification-guide-1.png)
      
    * Indentation: In TEAMMATES, we use 4 spaces in place of tabs for indentations.
-     Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
-     * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
-     * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`
+   * Trailing whitespace: all forms of trailing whitespace is disallowed, blank lines included.
+   * Go to `Eclipse → Preferences` and configure for all the languages used in TEAMMATES:
+     * Java:
+        * `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`
+        * `Java → Editor → Save Actions → Additional actions → Configure → Remove trailing whitespace → All lines`
```

Comment:
One more step:
...`Save Actions` -> `Perform the selected action on save` -> `Additional actions`...
How about other 3/4 actions? Do we need this? If not, we need to tell the developers to unclick it.
```
- Add missing '@Override' annotations
- Add missing '@Override' annotations to implementations of interface methods
- Add missing '@Deprecated' annotations
- Remove unnecessary casts
- Remove trailing white spaces on all lines

```